python/ray/dashboard/modules/data/tests/test_data_head.py

```diff
@@ -65,11 +65,11 @@ def test_get_datasets():
     assert len(operators) == 2
     op0 = operators[0]
     assert sorted(op0.keys()) == sorted(OPERATOR_SCHEMA)
-    assert op0["operator"] == "Input0"
+    assert op0["operator"] == "Input_0"
     assert op0["progress"] == 20
     assert op0["total"] == 20
     assert op0["state"] == "FINISHED"
-    assert operators[1]["operator"] == "ReadRange->MapBatches(<lambda>)1"
```

Comment:
Let's assert the shape of the whole serialized output

Reply:
@alexeykudinkin are the values of the whole object constant; all the fields are, but I'm not sure if those `ray_data_...` have fixed values appropriate as a test case

```
"state",
"progress",
"total",
"ray_data_output_rows",
"ray_data_spilled_bytes",
"ray_data_current_bytes",
"ray_data_cpu_usage_cores",
"ray_data_gpu_usage_cores",
```